core-concepts/streams/stream-names.md

```diff
@@ -25,21 +25,21 @@ A _category_ stream name does not have an ID. For example, the stream name for t
 
 ## Category Types
 
-Category stream names can be broken down into sub-categories, or _types_.
+Stream names can also have an optional list of _types_.
 
-The `command` type is a very common category type. It's used to indicate a stream that is used to transport _commands_ rather than _events_.
+The `command` type is one of the most common types associated with a stream. It's used to indicate a stream that is used to transport _commands_ rather than _events_.
 
 For example, a stream name for the stream that holds commands for the `account-123` entity would be named `account:command-123`.
 
-The category name is separated from the category type by the `:` character.
+The stream name is separated from the type list by the `:` character.
 
-Category types can be present in both entity stream names as well as category stream names.
+Types can be present in both entity stream names as well as category stream names.
 
 For example, the stream name for all commands for the `account` category would be `account:command`.
 
 ## Many Category Types
 
-A stream name can have more than one category type.
+A stream name can have more than one type.  The `+` symbol joins types when there are more than one.
```

Comment:
Again, I'm leery of using "join" here. It technically less clear than what the reality is. I maintain that if we had used a character like a comma, then the mind would not slip into seeing it as a "join". The "+" sign _separates_ the types. It's a _separator_.